### ISSUE TITLE
Installation fails on Mac

### ISSUE THREAD

**Issue by bgbg** [CAMeL-Lab/camel_tools#60]
running `pip-install camel-tools` fails on Mac in a freshly created Anaconda environment, and in other environments. I tested this with Python 3.7 and 3.8. Attached is the log file of my last install attempt

[install.log](https://github.com/CAMeL-Lab/camel_tools/files/6975965/install.log)

**Comment by owo** [comment]
This seems to be an installation issue with one of our dependencies.
Can you try running `pip install --upgrade pip`  before trying to reinstall camel_tools and let me know if that fixes the issue?
That usually fixes compilation issues for that particular dependency for me.

**Comment by bgbg** [comment]
Still fails
[install.log](https://github.com/CAMeL-Lab/camel_tools/files/6982618/install.log)
x
```bash
(base)  data % pip install --upgrade pip
Requirement already satisfied: pip in /Users/boris/opt/anaconda3/lib/python3.8/site-packages (21.1)
Collecting pip
  Downloading pip-21.2.4-py3-none-any.whl (1.6 MB)
     |████████████████████████████████| 1.6 MB 938 kB/s
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 21.1
    Uninstalling pip-21.1:
      Successfully uninstalled pip-21.1
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/boris/opt/anaconda3/include/python3.8/UNKNOWN
sysconfig: /Users/boris/opt/anaconda3/include/python3.8
WARNING: Additional context:
user = False
home = None
root = None
prefix = None
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
orange3-bioinformatics 4.3.1 requires resdk, which is not installed.
orange3-singlecell 1.4.0 requires fastdtw==0.3.2, but you have fastdtw 0.3.4 which is incompatible.
orange3-singlecell 1.4.0 requires xlrd~=1.2.0, but you have xlrd 2.0.1 which is incompatible.
Successfully installed pip-21.2.4
(base)  data % pip install camel-tools > ~/temp/install.log 2>&1
```

the log file is attached

**Comment by owo** [comment]
I tried recreating your environment as best as I could, however you are currently running an older macOS version which I can't replicate at this time.

I am using:
- MacOS 11.4 for x86_64
- Xcode 12.5.1 
- anaconda3-2021.05 (installed via [pyenv](https://github.com/pyenv/pyenv))

I ran the following without issues:

```
conda create -n test_ct python=3.8
conda activate test_ct
pip install camel_tools
```

Mac has always been problematic when it comes Python and the best solution I have found so far is to manage Python installations via [pyenv](https://github.com/pyenv/pyenv).

The only other potential fix I can think of is to update Xcode and/or run `sudo xcode-select --install`.
